### PR TITLE
Drop CocoaPods support

### PR DESCRIPTION
**Pull request by tomhai**
Removes CocoaPods releasing related functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,23 +10,9 @@ Allihoopa SDK for iOS
...
-end
-```
-
-And then running the following command to download the dependency:
-
-```bash
-pod install
-```
-
 If you use [Carthage], you instead add this SDK to your `Cartfile`:
```
Comment (Boerworz) [inline README.md]: "instead" should be removed.

File: README.md [hunk 2]
```diff
@@ -81,9 +66,6 @@ Look in the [SDKExample] folder for instructions how to work on this SDK.
 
 # API documentation
```
Comment (Boerworz) [inline README.md]: Perhaps a better heading would be "Usage" since there's no proper documentation now?

File: README.md (hunk 1 above)
Comment (tomhai) [inline README.md]: ok

File: README.md (hunk 2 above)
Comment (tomhai) [inline README.md]: ok

**Pull request merged**